generic/ras.py

```diff
@@ -40,6 +40,14 @@ def setUp(self):
         architecture = os.uname()[4]
         if "ppc" not in architecture:
             self.skip("supported only on Power platform")
+        if 'PowerNV' in process.system_output('grep ^platform /proc/cpuinfo'):
+            nv_flag = False
+            for cmd in ['lsmcode', 'lsprop', 'nvram', 'ppc64_cpu']:
```

Comment:
Let me see if I got this correctly... It means that on `PowerNV` platform you will only run these tests: 'lsmcode', 'lsprop', 'nvram', 'ppc64_cpu'?